### DISCUSSION TITLE
Change default pt speed / distance in config

### DISCUSSION THREAD

**Discussion by FusselTV** [QuantumEntangledAndy/neolink#232] [Q&A]
I have a Reolink E1 and noticed when I use the MQTT PT buttons in HomeAssistant, the movement is higher than in the Reolink app. Can I change the default 32 speed in the config with an entry to have finer control for the HomeAssistant buttons?

**Comment by QuantumEntangledAndy** [comment]
Simply create the command you want in your home assistant config

https://www.home-assistant.io/integrations/button.mqtt/

In this case you'd want to make a button that posts the direction with another value than 32
